config/routes.rb

```diff
@@ -4,12 +4,16 @@
     get "/login", to: "sessions#new"
     post "/login", to: "sessions#create"
     delete "/logout", to: "sessions#destroy"
-    resources :users, only: :show
+    resources :users do
+      member do
+        get "order_history"
```

Comment:
```suggestion
        get "orders"
```
cần j history cụ

mà nếu đã đổi như chị thì cần quái j member đồ

chị lại thích dùng nested recourse hơn ấy =))
```
resources :users do
  resources :orders, only: :index
end
```